web3deployment/deployerv3.js

```diff
@@ -0,0 +1,473 @@
+  // set proxy of network
+  console.log("set proxy of network");
+  await sendTx(networkContract.methods.setKyberProxy(proxyAddress));
+
+  // set whitelist
+  console.log("set whitelist address");
+  await sendTx(networkContract.methods.setWhiteList(whitelistAddress));
+
+  // set expected rate contract
+  console.log("set expected rate address");
+  await sendTx(networkContract.methods.setExpectedRate(expectedRateAddress));
+
+  // set fee burner contract
+  console.log("set fee burner contract");
+  await sendTx(networkContract.methods.setFeeBurner(feeBurnerAddress));
+
+  // add operator to network
+  console.log("network - set temp operator");
+  await sendTx(networkContract.methods.addOperator(sender));
+  console.log("fee burner - set temp operator");
+  await sendTx(feeBurnerContract.methods.addOperator(sender));
+  console.log("expected rate - set temp operator");
+  await sendTx(expectedRateContract.methods.addOperator(sender));
+
+  // set permissionless orderbook lister as network operator
+  console.log("network - set permissionless lister");
+  await sendTx(networkContract.methods.addOperator(permissionlessOrderbookReserveListerAddress));
+
+  // add reserve to network
+  console.log("Add reserves to network");
+  for(let i = 0 ; i < reserveDataArray.length ; i++) {
+    const reserve = reserveDataArray[i];
+    console.log(`Adding reserve ${reserve.address}`);
+    await sendTx(networkContract.methods.addReserve(reserve.address,false));
+    const tokens = reserve.tokens;
+    console.log("list reserve tokens");
+    for(let j = 0 ; j < tokens.length ; j++) {
+      token = tokens[j];
+      await sendTx(networkContract.methods.listPairForReserve(reserve.address,token.address,token.ethToToken,token.tokenToEth,true));
+    }
+    console.log("set reserve fees");
+    await sendTx(feeBurnerContract.methods.setReserveData(reserve.address,
+                                                          reserve.fees,
+                                                          reserve.wallet));
+  }
+
+  // set params
+  console.log("network set params");
+  await sendTx(networkContract.methods.setParams(maxGasPrice,
+                                                 negDiffInBps));
+
+  console.log("network enable");
+  await sendTx(networkContract.methods.setEnable(true));
+  
+  console.log("network - remove temp operator");
+  await sendTx(networkContract.methods.removeOperator(sender));
+  await setPermissions(networkContract, networkPermissions);
+
+  // burn fee
+  console.log("set KNC to ETH rate");
+  await sendTx(feeBurnerContract.methods.setKNCRate());
+  console.log("set tax fees bps");
+  await sendTx(feeBurnerContract.methods.setTaxInBps(taxFeesBps));
+  if(taxWalletAddress != '' && taxWalletAddress != 0) {
+    console.log("set tax wallet address");
+    await sendTx(feeBurnerContract.methods.setTaxWallet(taxWalletAddress));
+  }
+
+  //set wallet fees for non-standard fees
+  for(let i = 0 ; i < walletDataArray.length; i++) {
```

Comment:
great.
did u try reading events?
did u find some of those wallets with fee > 30

Reply:
Yup, I did. I didn't add that script... should I?